action-server/src/config.ts

```diff
@@ -0,0 +1,37 @@
+export interface Config {
+  AERIE_DB: string;
+  AERIE_DB_HOST: string;
+  AERIE_DB_PORT: string;
+  ACTION_DB_USER: string;
+  ACTION_DB_PASSWORD: string;
+  ACTION_LOCAL_STORE: string;
+  ACTION_WORKER_NUM: string;
+  ACTION_MAX_WORKER_NUM: string;
+  HASURA_GRAPHQL_ADMIN_SECRET: string;
+  LOG_FILE: string;
+  LOG_LEVEL: string;
+  MERLIN_GRAPHQL_URL: string;
+  PORT: string;
+  STORAGE: string;
+}
+
+export const configuration = (): Config => {
+  const { env } = process;
+
+  return {
+    AERIE_DB: env.AERIE_DB ?? "aerie",
+    AERIE_DB_HOST: env.AERIE_DB_HOST ?? "postgres",
+    AERIE_DB_PORT: env.AERIE_DB_PORT ?? "5432",
+    ACTION_DB_USER: env.ACTION_DB_USER ?? "postgres",
+    ACTION_DB_PASSWORD: env.ACTION_DB_PASSWORD ?? "password",
```

Comment:
Credentials should always default to an empty string (see L30 of this file):
```suggestion
    ACTION_DB_USER: env.ACTION_DB_USER ?? "",
    ACTION_DB_PASSWORD: env.ACTION_DB_PASSWORD ?? "",
```

Reply:
Removed the user and pasword from the configuration